src/med_bench/mediation.py

```diff
@@ -103,162 +107,46 @@ def mediation_IPW(y, t, m, x, w, z, trim, logit, regularization=True, forest=Fal
     crossfit integer, default 0
              number of folds for cross-fitting
 
-    clip    float
+    clip    float, default 0.01
             limit to clip for numerical stability (min=clip, max=1-clip)
+
+    calibration boolean, default False
+             whether to use a calibration with cross validation on existing classifiers
+
+    calibration_method  str, default sigmoid
+            calibration mode; for example using a sigmoid function
```

Comment:
`calibration_boolean` can be removed by encoding it through  calibration_method=False`

Reply:
This has been addressed in 5c5d7c0b812d2aa64f6ae01e519138b341c2f8a5 .